packages/client/src/hooks/useLogin.ts

```diff
@@ -0,0 +1,17 @@
+import { fetchUser, login } from "@store/actions/AuthActionCreators"
+import { useAppDispatch } from "@store/index"
+import { LoginData } from "@store/types"
+import { useNavigate } from "react-router-dom"
+
+export const useLogin = () => {
+  const navigate = useNavigate()
+  const dispatch = useAppDispatch()
+
+  return (data: LoginData) => {
+    dispatch(login(data)).then((res) => {
+      if(res.payload ==='OK'){
+        (navigate('/'), dispatch(fetchUser()))
```

Comment:
nit: Очень странное форматирование 🤔 

```suggestion
        navigate('/')
        dispatch(fetchUser())
```

Reply:
поправила